communitytoolkit/maui/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: .NET Multi-platform App UI (.NET MAUI) Community Toolkit
+author: bijington
+ms.author: bramin
```

Comment:
You can drop the ms.author metadata from every article. This will be configured at the repo level.